internal/transport/keepalive_test.go

```diff
@@ -340,19 +314,12 @@ func (s) TestKeepaliveClientClosesWithActiveStreams(t *testing.T) {
 	}
 	defer conn.Close()
 
-	ctx, cancel := context.WithTimeout(context.Background(), defaultTestTimeout)
-	defer cancel()
-	// Create a stream, but send no data on it.
-	if _, err := client.NewStream(ctx, &CallHdr{}); err != nil {
-		t.Fatalf("client.NewStream() failed: %v", err)
+	if err := pollForHealthyStream(client); err != nil {
```

Comment:
Interesting...this function leaks a stream, which is why this test is equivalent to how it was before.  However, this is not obvious.  Just create the stream here directly, since its existence is important, and maybe make `pollForHealthyStream` cancel the stream it creates before returning?